outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/service/impl/TargetFileServiceImpl.java

```diff
@@ -130,6 +136,16 @@ private String targetFileName() {
     }
 
 
+    private void insertTargetFileAuditRecord(TargetFileNotification tfn, String status) {
+        if (tfn == null) {
+            fileAuditDataService.create(new AuditRecord(FileType.TargetFile, null, null, null, status));
+        } else {
+            fileAuditDataService.create(new AuditRecord(FileType.TargetFile, tfn.getFileName(), tfn.getRecordCount(),
+                    tfn.getChecksum(), status));
+        }
+    }
+
+
     private File createTargetFileDirectory() {
         File userHome = new File(System.getProperty("user.home"));
         File targetFileDirectory = new File(userHome, settingsFacade.getProperty(TARGET_FILE_DIRECTORY));
```

Comment:
I think they might be giving us an ftp location to place the file.  I think the File class will handle that but I'm not sure.